src/utilities.jl

```diff
@@ -240,27 +242,54 @@ function Base.show(io::IO, pca::PCA;
                 pca.corr ? "correlation" : "(relative) covariance",
                 " matrix")
         # only display the lower triangle of symmetric matrix
-        Base.print_matrix(io, round.(LowerTriangular(pca.covcor), digits=ndigitsmat))
+        printmat = round.(LowerTriangular(pca.covcor), digits=ndigitsmat)
+        if pca.rnames !== missing 
+            n = length(pca.rnames)
+            cv = string.(printmat)
+            dotpad = lpad(".", div(maximum(length, cv),2))
+            for i = 1:n, j = (i+1):n
+                cv[i, j] = dotpad
+            end
+            neg = startswith.(cv, "-")
+            if any(neg)
+                cv[.!neg] .= " ".* cv[.!neg]
+            end
+            # this hurts type stability, 
+            # but this show method shouldn't be a bottleneck
+            printmat = Text.([pca.rnames cv])
+        end
+        
+        Base.print_matrix(io, printmat)
         println(io)
     end
     if stddevs
-        println(io, "Standard deviations:")
+        println(io, "\nStandard deviations:")
         sv = pca.sv
         show(io, round.(sv.S, digits=ndigitsvec))
         println(io)
     end
     if variances
-        println(io, "Variances:")
+        println(io, "\nVariances:")
         vv = abs2.(sv.S)
         show(io, round.(vv, digits=ndigitsvec))
         println(io)
     end
-    println(io, "Normalized cumulative variances:")
+    println(io, "\nNormalized cumulative variances:")
     show(io, round.(pca.cumvar, digits=ndigitscum))
     println(io)
     if loadings
-        println(io, "Component loadings")
-        Base.print_matrix(io, round.(pca.loadings, digits=ndigitsmat))
+        println(io, "\nComponent loadings")
+        printmat = round.(pca.loadings, digits=ndigitsmat)
+        
+        if pca.rnames !== missing 
+            pclabs = [Text(""); Text.( "PC$i" for i in 1:length(pca.rnames))]
```

Comment:
Will there be an alignment problem if `length(pca.rnames) > 9`?

Reply:
I think the bigger alignment problem will be if the numerical column width (arising `ndigitsmat`) is less than `maximum(length, pca.rnames)`.  But `Base.print_matrix` is pretty smart when it comes to printing `String`/`Text`.